taker-frontend/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/semver": "^7.3.10",
     "@vitejs/plugin-react": "^1.3.2",
     "@vitejs/plugin-react-refresh": "^1.3.1",
+    "axios": "^0.27.2",
```

Comment:
👍 For some reason this reminds of [this](https://github.com/comit-network/comit-i/blob/8a8496931747fd283632ee18963ca75f082b195d/package.json#L17).

Reply:
Mhmm.. comit-i ... good memories 😅